package.json

```diff
@@ -5,16 +5,20 @@
   "author": "Brandon Keown <bkeown@blitz.io>",
   "description": "A simple helpful robot for your Company",
   "scripts": {
+    "coverage": "istanbul cover --include-all-sources --root lib _mocha -- --reporter nyan 'spec/**/*.spec.js'",
     "hubot": "hubot --adapter slack",
-    "test": "mocha --watch spec/**/*.spec.js --reporter min",
-    "coverage": "istanbul cover --include-all-sources --root lib _mocha -- --reporter nyan 'spec/**/*.spec.js'"
+    "lint": "eslint .",
+    "pre-commit-msg": "echo 'Pre-commit linting and unit testing' && exit 0",
+    "test": "mocha spec/**/*.spec.js --reporter min",
+    "test-watch": "mocha --watch spec/**/*.spec.js --reporter min"
```

Comment:
This is just a small thing, but I'm not sure that I would make the automated one the nice one (test has special semantics [npm test vs npm run test-watch]).  You can make the special named one like pre-commit-test, and use like the dot reporter for it.